transform_assumption.ml

```diff
@@ -32,11 +32,11 @@ let insert_checkpoints (func:afunction) =
           version=version.label;
           pos=checkpoint_label pc;
         } in
-        InsertBefore [Osr {label=checkpoint_label pc; cond=[]; target; map=osr};]
+        InsertBefore [Osr {label=checkpoint_label pc; cond=[]; target; var_map=osr; frame_maps=[]};]
     in
-    if pc = 0 then Unchanged else
```

Comment:
Why was it there in the first place :) I don't know.

Maybe it was because it could go wrong if the first instruction was a label. But since we forbid this now I don't see a reason for this case.